index.html

```diff
@@ -238,6 +248,7 @@ <h3>
         </h3>
         <pre class="idl">
           dictionary ShareData {
+            FrozenArray&lt;File&gt; files;
```

Comment:
I think this should be a `sequence`, no? We wouldn't expect a developer to pass in a frozen array (i.e., `{files: Object.freeze(filesArray)}`). 
```suggestion
            sequence&lt;File&gt; files;
```